#! /usr/bin/env python

PACKAGE='autorally_core'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("camera_trigger_frequency",     int_t,    0,    "FPS for camera triggering",      40,    0, 150)

exit(gen.generate(PACKAGE, "autorally_core", "camera_trigger_params"))
